lib/convert-import-to-call.js

```diff
@@ -2,19 +2,29 @@
 
 const {
     CallExpression,
-    Identifier,
+    Import,
 } = require('@babel/types');
 
 const setLiteral = require('./set-literal');
 
 module.exports = (path) => {
-    const {source} = path.node;
+    const {loc, source} = path.node;
     
     setLiteral(source);
     
-    const callNode = CallExpression(Identifier('import'), [
+    const calleeNode = Import();
```

Comment:
This is a very good point!

Reply:
Could you please tell me a little bit more about `loc` in `callee` why do you need it?

Reply:
I don't personally need it, just searched for all fixtures which have no `"loc"` and compared it to the babel ast:
![image](https://github.com/coderaiser/estree-to-babel/assets/55899582/39624001-44cd-4c25-a66f-f92a89a915e2)

it can also be removed again if you want

Reply:
OK, I see, all this have sense. Could you please add one test with `maryiah` parser and get back loc calculation for `Imports`.